src/Style/Input/CheckboxStyle.ts

```diff
@@ -69,3 +69,7 @@ export const CheckboxContainer = styled.div`
     }
   }
 `;
+
+interface CheckboxContainerProps {
```

Comment:
Could we place `CheckboxContainerProps` at `Checkbox.tsx` to centralize the interface of component props?

Therefore, https://github.com/glints-dev/glints-aries/pull/267/files#diff-c80f91432b71eee8dd792beba827c3cdR46 can extend from `CheckboxContainerProps` to avoid redeclaring `small`.